### PR TITLE
Add circleci index plugin

### LINKED ISSUE #267: Add new index plugin
The **drone-circleci** plugin allows to trigger circle-ci builds

- [Repo](https://github.com/wesleimp/drone-circleci)
- [Docs](https://github.com/wesleimp/drone-circleci/blob/master/DOCS.md)

### PR DESCRIPTION
**Pull request by wesleimp**
Closes #267

### CODE REVIEW

File: content/wesleimp/drone-circleci/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+version: '0.8'
```
Comment (appleboy) [inline content/wesleimp/drone-circleci/index.md]: remove this line.

**Comment by wesleimp** [comment]
@appleboy could you please review it again?

File: content/wesleimp/drone-circleci/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+date: 2020-03-05T00:00:00+00:00
+title: Circle-ci
```
Comment (tboerger) [inline content/wesleimp/drone-circleci/index.md]: You should name it `CircleCI` as this is the proper title for that.

File: content/wesleimp/drone-circleci/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+author: wesleimp
+tags: [ infrastructure, trigger, circleci ]
+logo: circleci.svg
+repo: wesleimp/drone-circleci
+image: wesleimp/drone-circleci
+---
+
+The Circle-ci plugin allows you to trigger builds automatically. The below pipeline configuration demonstrates simple usage:
```
Comment (tboerger) [inline content/wesleimp/drone-circleci/index.md]: Same here

**Pull request merged**